### PR TITLE
Feature/automated deployment docs

### PR DESCRIPTION
**Pull request by ray-millward-tessella**
Added docs to describe the deployment process.

### CODE REVIEW

File: docs/SMG/08-deployment.md [hunk 1]
```diff
@@ -1,5 +1,20 @@
 # Deployment
 
+## Doing a regular simple deployment
+
+For a simple deployment with confguration changes (or if those configuration changes can be made after deployment), there is a GitHub action for deploying the system.
```
Comment (andykitch) [inline docs/SMG/08-deployment.md:5]: confguration -> configuration

File: docs/SMG/08-deployment.md [hunk 2]
```diff
@@ -9,7 +24,31 @@ ssh -i {certificate pem file here} vectoratlasadmin@20.87.47.170
...
+cd ..
+chmod +x buildVersionFiles.sh
+./buildVersionFiles.sh
+
+cd Docker
+docker-compose build
+docker-compose up --detach
+```
+
+### Running database migrations
```
Comment (andykitch) [inline docs/SMG/08-deployment.md:44]: Do we still need this now it's part of the stack?

File: docs/SMG/08-deployment.md (hunk 1 above)
Comment (ray-millward-tessella) [reply docs/SMG/08-deployment.md]: done

File: docs/SMG/08-deployment.md (hunk 2 above)
Comment (ray-millward-tessella) [reply docs/SMG/08-deployment.md:44]: I've put in some extra text to make it clearer that this normally happens as part of the docker compose stack but said that if we need to run it manually then this is the command.

**Review by andykitch — APPROVED** [review]

**Pull request merged**